deploy/helm/kubecf/assets/operations/instance_groups/api.yaml

```
@@ -13,6 +13,14 @@
     docker: docker_app_lifecycle/docker_app_lifecycle.tgz
 {{- end }}
 
+{{if .Values.testing.brain_tests.enabled -}}
+# For brain tests, allow CC access to the insecure registry they will create.
+- type: replace
+  path: /instance_groups/name=api/jobs/name=cloud_controller_ng/properties/cc/diego/insecure_docker_registry_list?
+  value:
+  - "insecure-registry.tcp.((system_domain)):20005"
```

```suggestion
  path: /instance_groups/name=api/jobs/name=cloud_controller_ng/properties/cc/diego/insecure_docker_registry_list?/-
  value:
    "insecure-registry.tcp.((system_domain)):20005"
```
This _should_ make it append to any existing lists.  Probably.  Maybe.  Hopefully.

---

Hm. Can trial. Will be tomorrow, current post-rebase trial is running and EOD when it is done.